cmd/openshift-install/waitfor.go

```diff
@@ -46,6 +46,8 @@ func newWaitForBootstrapCompleteCmd() *cobra.Command {
 
 			err = waitForBootstrapComplete(ctx, config, rootOpts.dir)
 			if err != nil {
+				logrus.Info("Use the following commands to gather logs from the cluster")
+				logrus.Info("> openshift-install --dir <root-dir> gather bootstrap")
```

Comment:
nit: drop the `>`

and `openshift-install gather bootstrap --help` so that people see the flags that can be used.

Reply:
Done.